odoo/addons/base/tests/test_qweb.py

```diff
@@ -1342,8 +1342,8 @@ def test_no_empty_lines(self):
             'arch_db': self.arch
         })
         rendered = self.env['ir.qweb']._render(t.id)
-        self.assertFalse(re.compile('^\s+\n').match(rendered))
-        self.assertFalse(re.compile('\n\s+\n').match(rendered))
+        self.assertFalse(re.compile('^\\s+\n').match(rendered))
+        self.assertFalse(re.compile('\n\\s+\n').match(rendered))
```

Comment:
`\n` is recognized by the regexp engine as well
```suggestion
        self.assertFalse(re.compile(r'^\s+\n').match(rendered))
        self.assertFalse(re.compile(r'\n\s+\n').match(rendered))
```